package.json

```diff
@@ -59,11 +63,11 @@
     "webpack-visualizer-plugin": "0.1.11"
   },
   "peerDependencies": {
-    "lru-cache": "^5.1.1",
-    "vue": "^2.5.22",
-    "vue-i18n": "^8.7.0",
-    "vue-i18n-extensions": "^0.2.0",
-    "vue-server-renderer": "^2.5.22"
+    "lru-cache": "5.1.1",
+    "vue": "2.5.22",
+    "vue-i18n": "8.7.0",
+    "vue-i18n-extensions": "0.2.0",
+    "vue-server-renderer": "2.5.22"
```

Comment:
Sorry, this is my bad, I forgot to mention that for `peerDependencies`, we _should_ use `^` prefixes so that consuming apps are free to update those freely.  This is moreso specifying that the client apps needs it's own depdndency meeting this looser rule.  Above in `dependencies` and `devDependencies` we want to be strict and lock-into a version since those are the installed dependencies of this package